### PR TITLE
IM, JR | Clear Storage

### LINKED ISSUE #45: Create a Reset Button Screen on Day 4
On Day 4 (the day after the study is completed), we need to create a screen with a reset button that clears local storage and sets the recording day to 1. This is a button for the entire study to be reset

### PR DESCRIPTION
**Pull request by jrosen081**
- Add an episode clear screen at day 4
"AVOCADO!"

### CODE REVIEW

File: screens/ClearEpisodeScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import { useMainNavigation } from "../hooks/useMainNavigation";
+import { NavigationScreens } from "../navigation/MainNavigationContext";
+import { continueButtonStyle } from "../utils/StylingUtils";
+
+export const ClearEpisodeScreen = () => {
+  const mainNavigation = useMainNavigation();
+  const [password, setPassword] = useState("");
+
+  const validatePassword = async () => {
+    if (password === "aVocado!") {
```
Comment (dfarooq610) [inline screens/ClearEpisodeScreen.tsx:20]: Nice

File: screens/ClearEpisodeScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      await AsyncStorage.clear();
+      mainNavigation.navigate({ type: NavigationScreens.onboarding });
```
Comment (dfarooq610) [inline screens/ClearEpisodeScreen.tsx:22]: This is more of a general comment:

Ok so we clear storage and redirect ourselves to the onboarding screen to reset the study. What I want to ensure is that we don't actually start counting recording days again.

If I reset today and I ship the device out to the participant in two days, we need to ENSURE that when the participant gets the iPad and uses the app it is still day 1.

If we don't already ensure this, the fix would probably be writing the start day of the study into local storage once we make past the initial onboarding screen of the app

**Review by dfarooq610 — APPROVED** [review]
Looks Amazing To Me

File: screens/ClearEpisodeScreen.tsx (hunk 2 above)
Comment (jrosen081) [reply screens/ClearEpisodeScreen.tsx:22]: That is a good point. I am unsure of the exact behavior as of now in regards to the first day.

File: app.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
     "orientation": "portrait",
     "icon": "./assets/images/icon.png",
     "scheme": "myapp",
-    "userInterfaceStyle": "automatic",
+    "userInterfaceStyle": "light",
```
Comment (zefwang) [inline app.json:9]: Are we just assuming/forcing everyone to use light mode here?

**Review by zefwang — APPROVED** [review]
lucifer got the money

File: app.json (hunk 3 above)
Comment (jrosen081) [reply app.json:9]: Why yes we are. Dark mode is for suckers (even though I am a big fan of dark mode but besides the point)

File: screens/ClearEpisodeScreen.tsx (hunk 2 above)
Comment (jrosen081) [reply screens/ClearEpisodeScreen.tsx:22]: What a ridiculous question, it was ezpz to figure this one out cuz of Iman and my MASSIVE BRAIN

**Pull request merged**